src/source/core/Tasks.bs

```diff
@@ -112,6 +112,22 @@ namespace mc.tasks
       mc.tasks.observeNodeField(task, fieldName, callback, sendMode, true, scope)
     end if
 
+    if asBoolean(globalObj.isTrackingTaskUsage)
+      mc.tasks.observeNodeField(task, "state", mc.tasks.onTaskStateChange, "both")
+      if globalObj.taskUsage = invalid
```

Comment:
I don't think this is as accurate as we want. .because we can actually use createTask and _not_ create the task (it's rare but it can happen). So I think all the tracking should happen in the callback, even for task start